tools/pubsys/Cargo.toml

```diff
@@ -41,6 +41,7 @@ serde_json = "1"
 simplelog = "0.12"
```

Comment:
I think it might be better to change the commit message to 
```
pubsys: added SSM parameter validation

"a short paragraph to describe what changes you have and those
changes will help to build canary something like that"
```